src/library-authoring/add-content/AddContentContainer.test.tsx

```diff
@@ -114,7 +114,7 @@ describe('<AddContentContainer />', () => {
       collectionId,
     );
     // Mocks for ComponentEditorModal to work in tests.
-    jest.spyOn(editorCmsApi, 'fetchImages').mockImplementation(async () => ( // eslint-disable-next-line
+    jest.spyOn(editorCmsApi, 'fetchImages' as keyof typeof editorCmsApi).mockImplementation(async () => ( // eslint-disable-next-line
```

Comment:
[inform]: The problem with the failed test is most likely related to [these changes](https://github.com/openedx/frontend-app-authoring/pull/1490), if you remove these lines the test will pass successfully

Reply:
It's fixed by this PR: https://github.com/openedx/frontend-app-authoring/pull/1516 

It was supposed to have merged by now but there is some problem with the CLA bot.

Reply:
Thanks a lot!
I noticed that we have a possible related problem on the Schedule and Details page
https://github.com/openedx/frontend-app-authoring/pull/1516#issuecomment-2487818248